### PR TITLE
Update policy

### PR DESCRIPTION
**Pull request by malancas**
#### Summary

Removing some small sections from the policy.

### CODE REVIEW

**Comment by kommendorkapten** [comment]
I reckon we should remove this section too:
https://github.com/sigstore/timestamp-authority/pull/251/files#diff-669237df67d62d453aa0fb50309ac55463d0fe7f5760b750d80981dd2805d273L424-L426

> The TSA shall make available to subscribers and relying parties its practice statement, and other relevant documentation, as necessary, to assess conformance to the time-stamp policy.
> 
> NOTE 2: The TSA is not generally required to make all the details of its practices public.

That section. 

My interpretation: 
I think it's concerning as if we open up the TSA to the public, any user would be a subscriber or relying party. Even if the TSA is not open to public use for requesting a time-stamp, any verifier is considered a relying party, and so may request the practice statement.

**Review by kommendorkapten — APPROVED** [review]
🚀

File: docs/tsa-policy.md [hunk 1]
```diff
@@ -356,27 +351,6 @@ NOTE: It is advisable that, when obtaining a time-stamp token, the subscriber
 verifies that the time-stamp token has been correctly signed and that the
 private key used to sign the time-stamp token has not been compromised.
 
-## 6.3. Relying Party Obligations
```
Comment (haydentherapper) [inline docs/tsa-policy.md:359]: Should we leave this section in but remove a reference to 7.1.2? It seems like this is still useful to include guidance for a relying pary

File: docs/tsa-policy.md [hunk 2]
```diff
@@ -421,17 +395,6 @@ In particular:
   external organizations supporting the TSA services including
   the applicable policies and practices.
 
-- The TSA shall make available to subscribers and relying parties its
```
Comment (haydentherapper) [inline docs/tsa-policy.md:424]: I think this and NOTE 2 is fine to leave - "Make available documentation, as necessary", and the note says "we might not". You could change "shall" to "may" if you want to make it clearer that it's optional.

For example, if the TSA were to log timestamps to a transparency log, that should be publicly documented for auditability.

File: docs/tsa-policy.md (hunk 1 above)
Comment (kommendorkapten) [reply docs/tsa-policy.md:359]: Yeah, makes sense.

File: docs/tsa-policy.md (hunk 2 above)
Comment (kommendorkapten) [reply docs/tsa-policy.md:424]: Changing it to *may* is fine!

**Review by haydentherapper — APPROVED** [review]

**Review by kommendorkapten — APPROVED** [review]

**Pull request merged**